ios/README.md

```diff
@@ -5,6 +5,21 @@ Xcode
 
 ### Compiling from Command line
 To build Keyman for iOS, please run the build.sh build script within this folder.
-You will need to either run with the command-line argument `-no-codesign` or obtain the necessary security permissions and certificate data as part of our core development team to complete the build for the final app.
+You will need to run with the command-line argument `-no-codesign` or establish alternate code-signing permissions to complete the build for the final app.
 
 To run the app within a simulator after build.sh is completed, "Run" the Keyman scheme/target within Xcode.
+
+### Running the app on a physical device
+In order to test any code changes on a physical device, Apple requires that an app be code-signed.  The quickest way to handle this for personal testing:
+
+- Open Xcode and the settings for the Keyman target.
+- Establish personal code-signing permissions.
+  - Go to the **General** tab and enable "Automatically manage signing" and select your "Personal Team."
+  - Afterward, swap to the **Capabilities** tab and *disable* "App Groups."
```

Comment:
If you disable App Groups, will you be able to install keyboards and make them available to the system keyboard?

Reply:
Good catch.  You can add them to the app, but not the system keyboard.

Reply:
Interestingly, the "Get Started" item will not report the "install languages" op as successful when languages are downloaded if the system keyboard can't access them.